fastlane/actions/get_latest_github_release_name.rb

```diff
@@ -0,0 +1,105 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      GET_LATEST_GITHUB_RELEASE_INFO = :GET_LATEST_GITHUB_RELEASE_INFO
+    end
+
+    class GetLatestGithubReleaseNameAction < Action
+      def self.run(params)
+        UI.message("Getting latest release on GitHub (https://api.github.com/#{params[:repo_name]})")
+
+        GithubApiAction.run(
+          server_url: "https://api.github.com",
+          api_token: ENV["GITHUB_API_TOKEN"],
+          api_bearer: nil,
+          http_method: 'GET',
+          path: "repos/#{params[:repo_name]}/releases",
+          error_handlers: {
+            404 => proc do |result|
+              UI.error("Repository #{params[:repo_name]} cannot be found, please double check its name and that you provided a valid API token (if it's a private repository).")
+              return nil
+            end,
+            401 => proc do |result|
+              UI.error("You are not authorized to access #{params[:repo_name]}, please make sure you provided a valid API token.")
+              return nil
+            end,
+            '*' => proc do |result|
+              UI.error("GitHub responded with #{result[:status]}:#{result[:body]}")
+              return nil
+            end
+          }
+        ) do |result|
+          json = result[:json]
+          highest = json.last
```

Comment:
Nitpick. Don't think it's an issue, but we can throw an error if there are no releases.